qoqo/src/operations/multi_qubit_gate_operations.rs

```diff
@@ -64,3 +64,314 @@ pub struct MultiQubitZZ {
     /// The angle of the multi qubit Molmer-Sorensen gate.
     theta: CalculatorFloat,
 }
+
+// /// Module containing the CallDefinedGate class.
+// #[cfg(feature = "unstable_operation_definition")]
+// #[pymodule]
+// fn call_defined_gate(_py: Python, module: &Bound<PyModule>) -> PyResult<()> {
+//     module.add_class::<CallDefinedGateWrapper>()?;
+//     Ok(())
+// }
+
+/// The gate to be replaced by a gate defined with GateDefinition gate.
+/// The gate applies a gate previously defined by GateDefinition with the name gate_name.
+///
+/// Args:
+///     gate_name (str) : The name of the called defined operations.
+///     qubits (Vec<usize>) : The qubits that for this call replace the qubits in the internal definition of the called gate
+///                           (get replaced in order of apppearance in gate defintion).
+///     free_parameters (Vec<CalculatorFloat>) : List of float values that replace the free parameters in the internal defintion of the called gate
+///                                             (get replaced in order of apppearance in gate defintion).
+#[cfg(feature = "unstable_operation_definition")]
+#[pyclass(name = "CallDefinedGate", module = "qoqo")]
+#[derive(Debug, Clone, PartialEq)]
+pub struct CallDefinedGateWrapper {
```

Comment:
How come the macros can't be used here?

Reply:
It was because of the free_parameters that is a `Vec<CalculatorFloat>`, it worked when I used f64 but stopped when I switched to CalculatorFloat

Reply:
Alright, we will discuss this in a meeting.